docs/extensions-plugins/cli/commands.mdx

```diff
@@ -72,13 +72,23 @@ Lists all accessible AI models.
 pieces list models
 ```
 
-### `open`
-Opens an asset from a list or search. If only "open" is used, it opens the most recent asset.
+#### open (deprecated)
+Opens an asset from a list or search. If only "open" is used, it opens the most recent asset. This also creates a link to the asset's code.
 
 ```bash
-pieces open [ITEM_INDEX]
+pieces open [ITEM_INDEX] [-e]
 ```
 
+```-e``` is an optional flag . It Opens the asset in the configured external editor. Editor of choice can be configured using config command.
```

Comment:
The e flag work in the list not in the open
It is `list assets -e` to open the asset in the editor

Reply:
got it . fixed the change in commit [52568c5](https://github.com/pieces-app/documentation/pull/538/commits/52568c542135dcd2361d1486959019f2bc3d768a)